file_watcher/utils.py

```diff
@@ -0,0 +1,13 @@
+"""
+Contains general utilities for the file_watcher module
+"""
+import logging
+import sys
+
+stdout_handler = logging.StreamHandler(stream=sys.stdout)
+logging.basicConfig(
+    handlers=[stdout_handler],
+    format="[%(asctime)s]-%(name)s-%(levelname)s: %(message)s",
+    level=logging.INFO,
+)
+logger = logging.getLogger(__name__)
```

Comment:
Why is the logger being assigned here and imported in other places? as opposed to being assigned on a per module basis allowing the name to change etc.

Reply:
So I basically missed the point where the logger generates it's name using this, due to the size of the project, I'd be happy to replace __name__ with "FileWatcher".